configs/defs/generic_linux.textproto

```diff
@@ -13532,3 +13612,840 @@ benchmark_configs: {
+      "/sbin/aureport p+i+n+u+g+s+b+acl+xattrs+sha512\n"
+      "/sbin/autrace p+i+n+u+g+s+b+acl+xattrs+sha512\n"
+      "/sbin/augenrules p+i+n+u+g+s+b+acl+xattrs+sha512\n"
+      "```\n"
+    cis_benchmark: {
+      profile_level: 2
+      severity: MEDIUM
+    }
+    scan_instructions:
+      "generic:{check_alternatives:{"
+      "    file_checks:{"
+      "      files_to_check:{single_file:{path:\"/etc/aide/aide.conf\"}}"
+      "      content_entry:{"
+      "        match_type: ALL_MATCH_ANY_ORDER"
+      "        match_criteria:{"
+      "          filter_regex: \"/sbin/(auditctl|auditd|ausearch|aureport|autrace|augenrules)\\\\s+p\\\\+i\\\\+n\\\\+u\\\\+g\\\\+s\\\\+b\\\\+acl\\\\+xattrs\\\\+sha512.*\""
+      "          expected_regex: \"/sbin/(auditctl|auditd|ausearch|aureport|autrace|augenrules)\\\\s+p\\\\+i\\\\+n\\\\+u\\\\+g\\\\+s\\\\+b\\\\+acl\\\\+xattrs\\\\+sha512.*\""
+      "        }"
+      "      }"
+      "    }"
+      "}}"
+  }
+}
+
+benchmark_configs: {
+  id: "system-disabled-when-audit-logs-full"
+  compliance_note: {
+    version: { cpe_uri: "cpe:/o:debian:debian_linux:12" version: "1.1.0" benchmark_document: "CIS Debian Linux 12" }
+    version: { cpe_uri: "cpe:/o:canonical:ubuntu_linux:22.04" version: "2.0.0" benchmark_document: "CIS Ubuntu 22.04" }  
+    title: "Ensure system is disabled when audit logs are full"
+    description: "The auditd daemon can be configured to halt the system when the audit logs are full."
+    rationale:
+      "In high security contexts, the risk of detecting unauthorized access or "
+      "nonrepudiation exceeds the benefit of the system's availability."
+    remediation:
+      "Set the following parameters in /etc/audit/auditd.conf: \n"
+      "```space_left_action = email \n"
+      "action_mail_acct = root \n"
+      "admin_space_left_action = halt```"
+    cis_benchmark: {
+      profile_level: 2
+      severity: LOW
+    }
+    scan_instructions:
+      "generic:{"
+      " check_alternatives:{"
+      "  file_checks:{"
+      "    files_to_check:{single_file:{path:\"/etc/audit/auditd.conf\"}}"
+      "      content_entry:{"
+      "        match_type: ALL_MATCH_ANY_ORDER"
+      "        match_criteria: {"
+      "          filter_regex: \"\\\\s*space_left_action(\\\\s|=).*\""
+      "          expected_regex: \"space_left_action\\\\s*=\\\\s*email\""
+      "        }"
+      "        match_criteria: {"
+      "          filter_regex: \"\\\\s*action_mail_acct(\\\\s|=).*\""
+      "          expected_regex: \"action_mail_acct\\\\s*=\\\\s*root\""
+      "        }"
+      "        match_criteria: {"
+      "          filter_regex: \"\\\\s*admin_space_left_action(\\\\s|=).*\""
+      "          expected_regex: \"admin_space_left_action\\\\s*=\\\\s*(halt|single)\""
+      "        }"
+      "      }"
+      "  }"
+      " }"
+      " check_alternatives:{"
+      "  file_checks:{"
+      "    files_to_check:{single_file:{path:\"/etc/audit/auditd.conf\"}}"
```

Comment:
Remove this check_alternative - this check should fail if there's no config file